apps/client/src/features/viewers/timer/Timer.tsx

```diff
@@ -176,9 +176,9 @@ export default function Timer(props: TimerProps) {
           now={time.current}
           complete={totalTime}
           normalColor={viewSettings.normalColor}
-          warning={eventNow?.timeWarning ?? 0}
+          warning={eventNow!.timeWarning}
```

Comment:
do we have to do this?

I wonder if the type could be simply
`warning?: number;`
if we want to save it as nullable is also ok, just one more thing to check

Reply:
We need to check the eventNow here because it's possibly null.

Reply:
in that case you cant use the null assertion and just pass `warning={eventNow?.timeWarning}` so that it will be undefined if there is no event now